node/config.go

```diff
@@ -20,6 +22,13 @@ type Config struct {
 	Core core.Config
 	P2P  p2p.Config
 	RPC  rpc.Config
+	// hash of the ExtendedHeader of the first block,
+	// this is necessary to begin the header.Service as
+	// it requires a header to already be injected in the store
+	// before it can begin syncing, for verification purposes.
+	// TODO @renaynay + @Wondertan: this is a solution for Devnet
+	// and should be modified in the future.
```

Comment:
Are we tracking the shortcuts we are taking for devnet somewhere? 
There is also: 
- https://github.com/celestiaorg/celestia-core/pull/582
- https://github.com/celestiaorg/lazyledger-app/pull/149

I feel there was another one in node?